src/PAModel/EditorState/ControlState.cs

```diff
@@ -29,6 +29,8 @@ internal class ControlState
         // even though this can be re-derived from the existence of DynamicProperties
         public bool? HasDynamicProperties { get; set; }
 
+        public bool? AllowAccessToGlobals { get; set; }
```

Comment:
Is this used from ControlState, or was this testing?

Reply:
Not needed. Good catch. Thanks.

Reply:
This is now used.